sdk/src/main/java/com/mapbox/maps/renderer/widget/Widget.kt

```diff
@@ -40,4 +49,9 @@ abstract class Widget internal constructor() {
    * Get absolute rotation of widget in degrees.
    */
   abstract fun getRotation(): Float
+
+  @JvmSynthetic
+  internal fun setRepaintTrigger(doOnRepaint: (() -> Unit)?) {
```

Comment:
after looking a bit deeper and understanding what's happening here I guess it should be more like:
```
private var onTriggerRepaintAction: (() -> Unit)? = null

internal fun onTriggerRepaintAction(action: (() -> Unit)?) {
  onTriggerRepaintAction = action
}
```

cause now protected function is named `triggerRepaint` while here it's `setRepaintTrigger`, I didn't even connect them in the beginning...

Reply:
imho the suggested name is a bit more confusing for me, but as these are internal functions, I'd say we can leave it for now, unless someone else has strong opinion on the naming? cc @jush @yunikkk @ank27 @paulrenn67

Reply:
While it's indeed internal API and probably does not matter that much I'd probably renamed it to use `triggerRepaint` consistently, maybe 
```
private var triggerRepaintAction: (() -> Unit)? = null internal fun setTriggerRepaintAction(action: (() -> Unit)?) { triggerRepaintAction = action }

  protected fun triggerRepaint() {
    triggerRepaintAction?.invoke()
  }
```

Reply:
refactored as suggested